app/design/frontend/base/default/template/sync/recaptcha.phtml

```diff
@@ -0,0 +1,45 @@
+<?php
+if ($this->shouldCheckCaptcha()) : ?>
+<script type="text/javascript">
+    document.observe('dom:loaded', function() {
+        addRecaptcha();
+    });
+
+    function addRecaptcha() {
+        var newsletterForm = document.querySelector('#newsletter-validate-detail');
+
+        if (newsletterForm) {
+            window.smailyCaptchaSubmit = function() {
+                newsletterForm.submit();
+            };
+
+            var captchaDiv = document.createElement('div');
+            captchaDiv.classList.add('g-recaptcha');
+            captchaDiv.setAttribute('id', 'smaily-recaptcha');
+            newsletterForm.appendChild(captchaDiv);
+
+            var submitButton = newsletterForm.querySelector('#newsletter-validate-detail button[type="submit"]');
+            submitButton.addEventListener('click', function(event) {
+                event.preventDefault();
+                var validateForm = new VarienForm('newsletter-validate-detail');
+                if (validateForm.validator.validate()) {
+                    grecaptcha.execute();
+                };
+            });
+
+        }
+    };
+
+    function onLoadCallback() {
+        var newsletterForm = document.querySelector('#newsletter-validate-detail');
+        if (newsletterForm) {
+            grecaptcha.render('smaily-recaptcha', {
+            sitekey: "<?php echo $this->getCaptchaApiKey() ?>",
+            size : "invisible",
+            callback : "smailyCaptchaSubmit"
+        });
```

Comment:
Formatting: indentation. Should be one step deeper.

Reply:
Done db5f1a6025fc752c7739437d38b3644d68299912